docs/05-technical-reference/03-architecture/evnt-02-event-processing.md

```diff
@@ -5,7 +5,7 @@ type: Architecture
 The event processing and delivery flow uses the NATS server to process events and send them to subscribers.
 This diagram explains the event flow in Kyma, from the moment the Application sends an event, to the point when the event triggers the Function.
 
-![Eventing flow](./assets/eventing-flow.svg)
+![Eventing flow](./assets/evnt-processing-flow.svg)
```

Comment:
Please adjust the colors in the diagram to the guidelines (see the draft [here](https://github.com/kyma-project/community/blob/4e619fac18aadb1005927f1a5b77b5c0a576b684/guidelines/content-guidelines/02-diagrams.md#background)). Also, the `type` metadata is not needed here.